caampr-backend/src/main/java/com/jelistan/caampr/lambda/handler/GetGearListHandler.java

```diff
@@ -27,8 +29,10 @@ public GetGearListHandler(final GearProvider gearProvider) {
 
     @Override
     public APIGatewayProxyResponseEvent handleRequest(final APIGatewayProxyRequestEvent event, Context context)  {
+        GearListRequestAdaptor adaptor = new GearListRequestAdaptor();
```

Comment:
The adapter should be created in the dagger module (probably `ProviderModule`) and injected via the constructor

Reply:
gotcha will do